python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -15,6 +15,8 @@
 from ml_wrappers.model.pytorch_wrapper import WrappedPytorchModel
 from ml_wrappers.model.wrapped_classification_model import \
     WrappedClassificationModel
+from torchvision import transforms as T
```

Comment:
Seems we're taking a dependency on `torchvision`. Should this be included in the dependencies file, or (more preferably) we should instead mention somewhere in README that torchvision may be needed and hence download your own torch & torchvision module versions from pytorch's official website.

Reply:
Update: Looks like torchvision was already used in this repo before. Suggest following repo's principles on imports not directly included in dependencies.

```suggestion
try:
    from torchvision import transforms as T
except ImportError:
    module_logger.debug('Could not import torchvision, required if using' +
                        ' a vision PyTorch model')
```

Reply:
Good catch!! Thanks, will change